content/tagging/assigning_tags.md

```diff
@@ -13,25 +13,195 @@ further_reading:
+## Where to assign tags
+There are several places you can assign tags: using configuration files, [environment variables][80], the UI, [API][65], [DogStatsD][75], and inheriting from the [integrations][1]. It is recommended to rely on the configuration files and integration inheritance for most of your tagging needs.
 
-### Inheriting tags from an integration
+### Configuration Files
 
-The easiest method for assigning tags is to rely on the integration. Tags assigned to your Amazon Web Services instances, Chef recipes, and more are all automatically assigned to the hosts and metrics when they are brought in to Datadog.
+You can configure the host tags submitted by the Agent inside `datadog.yaml`. The tags for the [integrations][1] installed with the Agent are configured via yaml files located in the **conf.d** directory of the Agent install. To locate the configuration files, refer to [this page][59].
 
-The following [integrations][1] sources create tags automatically in Datadog:
+In YAML files, use a tag dictionary with a list of tags you want assigned at that level. Any tag you assign to the Agent is applied to every integration on that Agent's host.
 
-|                                         |                                                                                                                                                                                                                                                                                                                                               |
-| :-------------------------------------  | :------------------------------------------------------------------------------                                                                                                                                                                                                                                                               |
+Tag dictionaries have two different yet functionally equivalent forms:
+
+```
+tags: key_first_tag:value_1, key_second_tag:value_2, key_third_tag:value_3
+```
+
+or
+
+```
+tags:
+    - key_first_tag:value_1
+    - key_second_tag:value_2
+    - key_third_tag:value_3
+```
+
+It is recommended you assign tags as `key:value` pairs but simple tags are also accepted. See [how to use][61] tagging for more details.
+
+### Environment Variables
+
+When installing the containerized Datadog Agent, host tags can be set using the environment variable `DD_TAGS`. We automatically collect common tags from [Docker][77], [Kubernetes][78], [ECS][79], [Swarm, Mesos, Nomad and Rancher][77]. To extract even more tags, use the following options:
+
+| Environment Variable               | Description                                    |
+|------------------------------------|------------------------------------------------|
+| `DD_DOCKER_LABELS_AS_TAGS`         | extract docker container labels                |
```

Comment:
Descriptions should be capitalized and treated as a sentence. `extract docker container labels  ` => `Extract docker container labels.`